local_groups/forms.py

```diff
@@ -179,7 +179,7 @@ class Meta:
 
 class PasswordResetRequestForm(AuthPasswordResetRequestForm):
```

Comment:
This is also weird to be located in local groups app, I know it's out of scope but should be in organizing hub app.

Reply:
👍 

Can do while I'm in here.